docs/reference/parts.rst

```diff
@@ -0,0 +1,56 @@
+
+.. _ref_parts:
+
+***************
+Rockcraft parts
+***************
+
+.. rubric:: The main building blocks of a ROCK are *parts*.
+
+If this sentence sounds familiar, it's because **it is familiar**!
+Rockcraft parts are inherited from other existing Craft tools like
+`Snapcraft <https://github.com/snapcore/snapcraft>`_ and
+`Charmcraft <https://github.com/canonical/charmcraft>`_.
+
+Rockcraft *parts* go through the same lifecycle steps as Charmcraft and
+`Snapcraft parts <https://snapcraft.io/docs/parts-lifecycle>`_.
+
+The way the *parts*' keys and values are used in the *rockcraft.yaml* is exactly
+the same as in `snapcraft.yaml`_
+(`here <https://snapcraft.io/docs/adding-parts>`_ is how you define a *part*).
+
+Albeit being fundamentally identical to Snapcraft parts, Rockcraft parts
+actually offer some extended functionality and keywords:
+
+* **stage-packages**: apart from offering the well-known package installation
+  behavior, in Rockcraft the ``stage-packages`` keyword actually supports
+  chiseled packages as well (:ref:`learn more about Chisel
+  <chisel_explanation>`).
+  To install a package slice instead of the whole package, simply follow the
+  Chisel convention *<packageName>_<sliceName>*.
+
+
+Example
+.......
+
+.. _chisel-example:
+
+.. code-block:: yaml
+
+  parts:
+    chisel-openssl-binaries-only:
```

Comment:
this part's name is a bit misleading

Reply:
That bit was already there but in the spirit of Friday I updated it to `chisel-openssl` ;)